### PR TITLE
Collectd events

### PR DESCRIPTION
**Pull request by paramite**

### CODE REVIEW

File: plugins/handler/collectd-metrics/main.go [hunk 1]
```diff
@@ -79,9 +81,17 @@ func (c *collectdMetricsHandler) Handle(blob []byte, pf bus.MetricPublishFunc) {
 		err = c.writeMetrics(&cdmetric, pf)
 		if err != nil {
 			c.totalDecodeErrors++
+			if reportErrors {
```
Comment (pleimer) [inline plugins/handler/collectd-metrics/main.go:83]: Great idea! I don't see where reportErrors is defined however. Is this supposed to come in as a configuration? This also needs the bus.EventPublishFunction() to publish the event. This makes me believe we should have one type of handler that gets passed both the EventPublish and MetricPublish function

Comment (pleimer) [reply plugins/handler/collectd-metrics/main.go:83]: Ah I see what was happening here. The rebase just messed it up

**Comment by pleimer** [comment]
That last push also fixed the problem of the defaults being overwritten in the Print plugin

**Comment by pleimer** [comment]
The metrics path still runs as usual. I also ran the dummy-events with the collectd-events and ceilometer-events handler and printed them with the Print plugin. I have not tested actual storage into Elasticsearch yet. I will do that in the morning and then merge this

File: plugins/handler/collectd-metrics/main.go (hunk 1 above)
Comment (paramite) [reply plugins/handler/collectd-metrics/main.go:83]: Yup, that is rebase error :/ Yeah the bool should be removed now

**Comment by paramite** [comment]
Thanks Paul!

**Comment by pleimer** [comment]
Your welcome! Everything works now after I fixed a small mistake I made in the transfer process. Merging

**Pull request merged**